uaclient/clouds/identity.py

```diff
@@ -0,0 +1,50 @@
+import json
+
+from uaclient import exceptions
+from uaclient import clouds
+from uaclient import status
+from uaclient import util
+
+CLOUDINIT_RESULT_FILE = "/var/lib/cloud/data/result.json"
+
+
+@util.retry(FileNotFoundError, [1, 2])
```

Comment:
As discussed on IRC, this retry is no longer necessary (because the upstart job will ensure we never hit `FileNotFoundError` here, unless something very bad has happened and then waiting for a few seconds isn't going to help).

It also follows that we can drop the decorator defintion.

Reply:
+1, Let's drop that from this changeset before landing. I guess we should demo/aws-premium branch proper.

Reply:
This retry decorartor usage will disappear. But, we'll keep the definition because of the 2nd callsite in identity.